tests/test_spatialCell.py

```diff
@@ -254,10 +268,24 @@ def sortKey(a):
         self.assertEqual(sorted(ratings0, key=sortKey),
                          [cand.getCandidateRating() for cand in cell1])
 
+    def testStr(self):
+        expect = ("bbox=(minimum=(0, 0), maximum=(500, 500)), 6 cells\n"
+                  "Cell 0x0: bbox=(minimum=(0, 0), maximum=(259, 199)), ignoreBad=True, candidates=[]\n"
+                  "Cell 1x0: bbox=(minimum=(260, 0), maximum=(500, 199)), ignoreBad=True, candidates=[]\n"
+                  "Cell 0x1: bbox=(minimum=(0, 200), maximum=(259, 399)), ignoreBad=True, candidates=[]\n"
+                  "Cell 1x1: bbox=(minimum=(260, 200), maximum=(500, 399)), ignoreBad=True, candidates=[]\n"
+                  "Cell 0x2: bbox=(minimum=(0, 400), maximum=(259, 500)), ignoreBad=True, candidates=[]\n"
+                  "Cell 1x2: bbox=(minimum=(260, 400), maximum=(500, 500)), ignoreBad=True, candidates=[]")
+        self.assertEqual(str(self.cellSet), expect)
+
 
 class SpatialCellImageCandidateTestCase(unittest.TestCase):
 
     def setUp(self):
+        # To ensure consistency across tests: width/height are static members
+        # of SpatialCellImageCandidate, and tests can run in any order.
```

Comment:
> width/height are static members

:scream: :see_no_evil:

Reply:
I agree, but I'm not even thinking of attempting to fix that on this ticket.